spec/index.bs

```diff
@@ -2459,22 +2466,51 @@ that returns a new texture.
 <script type=idl>
 [Exposed=(Window, DedicatedWorker), SecureContext]
 interface GPUTexture {
+    readonly attribute GPUExtent3DDict size;
```

Comment:
WDYT of even inlining the size, so you'd get .width .height and .depthOrArrayLayers?

Reply:
We could do that, but if we keep it as a dict (or even a sequence or interface) then it becomes easier to pass around to other API calls. For example:

```
const depthTexture = device.createTexture({
  size: colorTexture.size,
  format: 'depth24plus',
  /* ... */
});
```